com.unity.render-pipelines.universal/Shaders/Utils/MaterialError.shader

```diff
@@ -0,0 +1,50 @@
+﻿Shader "Hidden/Universal Render Pipeline/MaterialError"
+{
+    SubShader
+    {
+        Pass
+        {
+            // Hybrid Renderer compatible error shader, which is used by Hybrid Renderer
+            // instead of the incompatible built-in error shader.
+
+            // TODO: Ideally this would be combined with FallbackError.shader, but it seems
+            // problematic because FallbackError needs to support SM2.0 and seems to use
```

Comment:
The fallback error has to work in built-in pipeline as of today as well, this way when a URP shader / material is used with built-in pipeline it appears pink instead of not rendering.

 Suggestion: Copy this Subshader to the top of the Hidden/Universal Render Pipeline/FallbackError shader so having both the Hybrid and SM 2.0 shader will work in all cases.

Reply:
Does using a different Subshader also make it possible to use SRP includes (that contain the DOTS instancing shader code) for the 4.5 subshader, and the built-in includes for the 2.0 subshader (which I think it is currently doing) ?